### DISCUSSION TITLE
Modeling Marine Plastic Spills "Pellet (Nurdle)" with OpenDrift

### DISCUSSION THREAD

**Discussion by kuohsienchang** [OpenDrift/opendrift#1397] [General]
Dear all,

I recently came across several articles on the marine plastic spill from the X-Press Pearl incident along Sri Lanka’s coast in 2021. This led me to wonder whether it’s possible to simulate "plastic pellet" dispersion using OpenDrift. My initial search suggests that there may not be a direct approach for this specific type of simulation.

Given this, could anyone suggest practical ways to configure OpenDrift parameters to model the movement of plastic pellets for a short period (~1-2 weeks), excluding considerations of UV-related decomposition? Any insights or recommendations would be greatly appreciated.

Thank you in advance for your insights.

Cheers,
Kuo-Hsien

**Comment by knutfrode** [comment]
I have earlier modeled plastic pellets (~5mm size) with the OceanDrift module without vertical mixing/entrainment, with good agreement with observations - indicating that they mostly float at surface. Horizontal drift was then surface current + 3% of wind - more or less equivalent to Stokes drift + 2% of wind.

It is also possible to include vertical mixing, but then a `terminal_velocity` must be prescribed for the elements, which can also be updated during simulation, as e.g. in the OpenOil model:
https://github.com/OpenDrift/opendrift/blob/master/opendrift/models/openoil/openoil.py#L912

But it is much easier if you can assume a fixed terminal velocity of the particles, although this can well be different for each particle.

Note that there is a PlastDrift module in OpenDrift, but this is basically a very thin wrapper around OceanDrift, which is very generic and configurable.
https://github.com/OpenDrift/opendrift/blob/master/opendrift/models/plastdrift.py
